MAINTAINERS.yml

```diff
@@ -1347,6 +1347,16 @@ Filesystems:
   labels:
     - "area: File System"
 
+"Filesystems: FatFs":
```

Comment:
Rename this as "FatFs reentrant support"

Reply:
Done.